src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -8238,6 +8321,47 @@ internal static bool WinIsSameFileSystemItem(string pathOne, string pathTwo)
             return false;
         }
 
+        internal static bool GetInodeData(string path, out UInt64 dev, out UInt64 inode)
```

Comment:
We do still use dev and inode. That's how we uniquely identify a file system object.